src/components/shared/buttons/SaveButton.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+    /**
+    * Our opinionated save button with loading and disabled state.
+    *
+    * *This is a functional component which has no state so does not need to define any props*
+    * @see https://vuejs.org/v2/guide/render-function.html#Functional-Components
+    *
+    * @example ./croud-save-button.md
+    */
+    export default {
+        functional: true,
+        name: 'croud-save-button',
+        render(h, context) {
```

Comment:
Uses render function as opposed to the functional templates that came in `Vue 2.5.0` as this is compatible with all of current remote repos (monrepo, croud-layout, etc)